### PR TITLE
Faq

### PR DESCRIPTION
**Pull request by KennethNielsen**
This PR adds a FAQ page to the docs with the first three questions and answers, which I thinks are among the most common.

Less importantly this also updates the docs with the newly decided upon support structure.

Questions/comments?

### CODE REVIEW

**Comment by ghcs27** [comment]
A FAQ page is certainly a good idea, and I haven't got any criticism of the implementation.

**Comment by stefankoegl** [comment]
Great idea :+1:

**Comment by KennethNielsen** [comment]
Thanks. Did either of you ( @ghcs27 @stefankoegl ) read though it for grammar and spelling etc?

**Comment by ghcs27** [comment]
Well, my English isn't that good, so I wouldn't quite trust my own review. That being said, here is what I noticed.

File: doc/faq.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.. _faq:
+
+Frequently Asked Questions
+==========================
+
+This page contains answer to a few commonly asked questions.
```
Comment (ghcs27) [inline doc/faq.rst]: Shouldn't it be 'answer**s**'?

File: doc/faq.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+At the face of it, *no*. Sonos® devices can only play music that is
+available on the network i.e. can be reached via a URI. So you have
+two options:
+
+1. You can share your local music folder onto the network and add it
+   to the Sonos® library as a part of your music collection, which can
+   then be searched, browsed and played with SoCo.
+2. You can cheat and make Python serve the files on the fly and play
+   them as URIs. The `play local files
+   <https://github.com/SoCo/SoCo/blob/master/examples/play_local_files/play_local_files.py>`_ example, shows one way in which this can be accomplished.
```
Comment (ghcs27) [inline doc/faq.rst]: That comma after 'example,' looks strange.

**Comment by KennethNielsen** [comment]
@ghcs27 I'm no English expert either, but it can never hurt with a couple of extra eyes on it, for content/grammar/spelling/typos etc. I incorporated the suggested changes. Thanks.

**Pull request merged**